version.txt

```diff
@@ -1,2 +1,2 @@
-0.1.7-SNAPSHOT
+1.0.0-SNAPSHOT
```

Comment:
Do we really want to go-to 1.0?  I would feel better staying in the 0. But that's just me

Reply:
Was trying to signify a major change as a result of bumping a major version of cass driver.

Happy to change / reflect that in whatever others feel makes sense.